src/app/_services/authentication.service.ts

```diff
@@ -71,10 +71,12 @@ export class AuthenticationService {
     }
 
     public getEmail(): string {
-        return this.email;
+        // return this.email;
+        return 'a@a';
     }
 
     public getRole() {
-        return this.role;
+        // return this.role;
+        return AuthenticationService.ROLE_ADMIN;
```

Comment:
Revertować te hacki?

Reply:
to wyrzuc, przeciez przychodzi rola

Reply:
Przychodzi... ale tak nie do końca... tzn. dalej jest problem że nie da się stworzyć nowego użytkownika, więc jak ktoś ma pustą bazę to trzeba robić takie hacki żeby wejść do aplikacji

Reply:
najlatwiej sobie dodac uzytkownika z adminem prosto do bazy